src/WordLists.jl

```diff
@@ -94,7 +96,11 @@ const LOOKUP_TABLE = Dict(
     "portuguese" => "pt",
     "português" => "pt",
     "por" => "pt",
-    "pt" => "pt"
+    "pt" => "pt",
+	"french" => "fr",
+	"français" => "fr",
+	"fre" => "fr",
+	"fr" => "fr"
```

Comment:
```suggestion
    "french" => "fr",
    "français" => "fr",
    "fre" => "fr",
    "fra" => "fr",
    "fr" => "fr"
```
Two changes: 4 spaces, not tabs; and "fra" is also a three letter abbreviation for French under [ISO639-2](https://www.loc.gov/standards/iso639-2/php/code_list.php) and I'm not aware of any other language "fra" could refer to so there shouldn't be any ambiguity issues down the line.